.github/workflows/ci.yml

```
@@ -16,8 +16,7 @@ jobs:
           - ubuntu-latest
           - macos-latest
           - windows-latest
-        python:
-          - '3.9'
+        python: [3.6, 3.7, 3.8, 3.9]
```

Let's just test oldest and newest Python versions

```suggestion
        python: [3.6, 3.9]
```